src/Uno.UI/UI/Xaml/IFrameworkElementImplementation.iOS.tt

```diff
@@ -30,8 +30,10 @@ using Windows.UI.Xaml.Automation;
 
 namespace <#= mixin.NamespaceName #>
 {
-    public partial class <#= mixin.ClassName #> : IFrameworkElement
+    public partial class <#= mixin.ClassName #> : IFrameworkElement, IXUidProvider
 	{		
+		string IXUidProvider.Uid { get; set; }
```

Comment:
should this only be for non-`FrameworkElements`?

Reply:
It avoids making two checks in the SetXName helper. Once we remove IFrameworkElement, we can remove this.